src/api/l_graphics_pass.c

```diff
@@ -680,11 +680,11 @@ static int l_lovrPassCopy(lua_State* L) {
 
   if (blob) {
     Buffer* buffer = luax_checktype(L, 3, Buffer);
-    uint32_t srcOffset = luax_optu32(L, 4, 0);
-    uint32_t dstOffset = luax_optu32(L, 5, 0);
+    graphics_size srcOffset = luax_optgraphics_size(L, 4, 0);
```

Comment:
`srcOffset` should be a `size_t` since it represents an offset into CPU memory.  It should be fine to copy a small range of bytes to a buffer, even if those bytes come from 100GB into a blob.